test/mint.js

```diff
@@ -318,4 +318,41 @@ describe('PodCastNFT', function () {
       )
     })
   })
+
+  describe('burn', () => {
+    it('can burn for admin', async () => {
+      const mintTx = await contract.mint(
+          'https://example.com/podcast.png',
+          ['Podcast Contributor'],
+          '10000',
+          owner.address
+      )
+      await mintTx.wait()
+
+      const transferTx = await contract['safeTransferFrom(address,address,uint256)'](
```

Comment:
I did not know this writing style. It is a good study. Thank you very much.

https://github.com/ethers-io/ethers.js/issues/1160#issuecomment-730176442